server/sonar-db-core/src/main/resources/org/sonar/db/version/schema-h2.ddl

```diff
@@ -449,7 +451,9 @@ CREATE TABLE "USERS" (
   "USER_LOCAL" BOOLEAN,
   "ONBOARDED" BOOLEAN NOT NULL,
   "CREATED_AT" BIGINT,
-  "UPDATED_AT" BIGINT
+  "UPDATED_AT" BIGINT,
+  "HOMEPAGE_TYPE" VARCHAR(40),
+  "HOMEPAGE_KEY" VARCHAR(64)
```

Comment:
I thought we agreed on homepage_value. We must clarify

Reply:
done